en/firefox_privacy_notice.md

```diff
@@ -83,64 +89,37 @@ Read the full documentation [here](https://firefox-source-docs.mozilla.org/toolk
-
-If you choose to enable "Improve the Firefox Suggest Experience," Firefox shares the following information to improve your browsing experience and to improve the service:
-
-* __Searches__: Firefox sends Mozilla what you type into the search bar and Mozilla may share that data with its [partners](https://support.mozilla.org/kb/firefox-suggest#w_who-are-mozillas-partners). 
-
-We take measures to limit our and our partners’ ability to identify you. [Learn more](https://support.mozilla.org/kb/firefox-suggest?as=u&utm_source=inproduc).
-
 ---
 
 ## If you use these features, Firefox will share data to provide you functionality and help us improve our products and services: {: #optional-features }
 
-### Firefox Accounts {: #firefox-accounts }
-
-* __Registration data__: Mozilla receives your email address and a hash of your password when you create a Firefox Account or sign-up to Join Firefox. You can choose to include a display name or profile image. Your email address is sent to our email vendor, Acoustic, which has its own [privacy policy](https://acoustic.com/privacy-notice/).
+### Mozilla Accounts {: #Mozilla-accounts }
 
-* __Location data__: For security purposes, we store the IP addresses used to access your Firefox Account in order to approximate your city and country. We use this data to send you email alerts if we detect suspicious activity, such as account logins from other locations.
+Mozilla receives registration, location, interaction and technical data when you create and use a Mozilla account. See the [Mozilla Accounts Privacy Notice](https://www.mozilla.org/privacy/mozilla-accounts/) for more information. 
 
-* __Interaction data__: We receive data such as your visits to the Firefox Accounts website, dashboards and menu preferences, what products and services you use in connection with your Firefox Account, and your interactions with our emails and SMS messages. We use this to understand your use of our products and services and to send you more useful Firefox Account Tips and in-product messages. Learn more about interaction data we collect in the [Glean Dictionary](https://dictionary.telemetry.mozilla.org/apps/accounts_frontend).
+### Review Checker {: #review-checker }
 
-* __Technical data__: To display which devices are synced to your Firefox Account and for security functionality, we store your device operating system, browser and version, timestamp, locale, and the same information for devices connected to your account. If you use your Firefox Account to log into other websites or services (such as AMO or Pocket), we receive the timestamp of those log-ins.
+Users in the US can opt-in to Review Checker, our guide to product reviews powered by Mozilla’s [Fakespot](https://www.fakespot.com/), which helps you understand how reliable product reviews are on [certain shopping sites](https://support.mozilla.org/en-US/kb/review-checker-review-quality). 
```

Comment:
```suggestion
Users in the US can opt in to Review Checker, our guide to product reviews powered by Mozilla’s [Fakespot](https://www.fakespot.com/), which helps you understand how reliable product reviews are on [certain shopping sites](https://support.mozilla.org/kb/review-checker-review-quality). 
```

Always drop en-US in links. This page will be localized, even if the first roll-out is US only.